exercises/list-ops/src/test/java/ListOpsTest.java

```diff
@@ -0,0 +1,364 @@
+        final List<Integer> list
+                = Collections.unmodifiableList(
+                        IntStream.range(0, 4)
+                                .boxed()
+                                .collect(Collectors.toList())
+                );
+        final List<Integer> actual = ListOps.concat(list, EMPTY_LIST);
+        final List<Integer> expected
+                = IntStream.range(0, 4)
+                        .boxed()
+                        .collect(Collectors.toList());
+
+        assertNotNull(actual);
+        assertFalse(actual.isEmpty());
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @Ignore
+    public void shouldConcatenateOneNonEmptyAndOneEmptyLists() {
+        final List<Integer> list
+                = Collections.unmodifiableList(
+                        IntStream.range(0, 4)
+                                .boxed()
+                                .collect(Collectors.toList())
+                );
+        final List<Integer> actual = ListOps.concat(EMPTY_LIST, list);
+        final List<Integer> expected
+                = IntStream.range(0, 4)
+                        .boxed()
+                        .collect(Collectors.toList());
+
+        assertNotNull(actual);
+        assertFalse(actual.isEmpty());
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @Ignore
+    public void shouldConcatenateTwoListsWithSameElements() {
+        final List<Integer> list1 = Collections.unmodifiableList(
+                IntStream.range(0, 4).boxed().collect(Collectors.toList())
+        );
+        final List<Integer> list2 = Collections.unmodifiableList(
+                IntStream.range(1, 6).boxed().collect(Collectors.toList())
+        );
+        final List<Integer> expected
+                = Stream.concat(list1.stream(), list2.stream())
+                        .collect(Collectors.toList());
+        final List<Integer> actual = ListOps.concat(list1, list2);
+
+        assertNotNull(actual);
+        assertFalse(actual.isEmpty());
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @Ignore
+    public void shouldConcatenateSeveralLists() {
+        final List<Integer> list1 = Collections.unmodifiableList(
+                IntStream.range(0, 4).boxed().collect(Collectors.toList())
+        );
+        final List<Integer> list2 = Collections.unmodifiableList(
+                IntStream.range(4, 8).boxed().collect(Collectors.toList())
+        );
+        final List<Integer> list3 = Collections.unmodifiableList(
+                IntStream.range(8, 12).boxed().collect(Collectors.toList())
+        );
+        final List<Integer> list4 = Collections.unmodifiableList(
+                IntStream.range(12, 16).boxed().collect(Collectors.toList())
+        );
+        final List<Integer> expected
+                = new ArrayList<>(list1);
+        expected.addAll(list2);
+        expected.addAll(list3);
+        expected.addAll(list4);
```

Comment:
🤔  what if we made it feasible to express `expected` literally?  For example, if we made each `listN` shorter, the concatenated list would fit on one line.